envs/conda-forge.yml

```diff
@@ -18,7 +18,7 @@ dependencies:
   - iris>=3.0,<3.1
   - netCDF4
   - numpy<1.21
-  - scipy
```

Comment:
@tjtg, 1.5.X or 1.6.X seem to work fine, 1.4.X produces a sphinx error. Suggesting pinning as <1.7?